src/components/MoreInfoModal/MoreInfoModal.tsx

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable react/jsx-props-no-spreading */
+/* eslint-disable react/no-array-index-key */
+import 'react-slideshow-image/dist/styles.css';
+
+import CloseIcon from '@mui/icons-material/Close';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import { Paper } from '@mui/material';
+import Button from '@mui/material/Button';
+import Modal from '@mui/material/Modal';
+import Typography from '@mui/material/Typography';
+import * as React from 'react';
+import { Slide } from 'react-slideshow-image';
+import photosHouse from 'src/assets/images/House.png';
+import photosMap from 'src/assets/images/Map.png';
+import photosNavLogo from 'src/assets/images/NavLogo.png';
+import photosNoPhoto from 'src/assets/images/nophoto.png';
+
+import FacilitiesList from './FacilitiesList';
+import styles from './MoreInfoModal.module.scss';
+import PropertyInformationList from './PropertyInformationList';
+
+const style = {
```

Comment:
It's OK if it stays here like this but to make declaring "inline" styles more consistent, you could insert the object containing the styles directly into the `sx` of the `Paper` that uses it and remove the variable `style`.